index.php

```diff
@@ -4,7 +4,7 @@
 
 	<div class="container">
         <div class="row">	
-            <div class="span8" id="content">
+            <main class="span8" id="content"> <!-- Why is this not showing up in the DOM on my browser? -->
```

Comment:
index.php is only used if no other page template is selected by WordPress's template hierarchy.

Reply:
Thank you! Would you mind if I made a note of that in this file?

Reply:
Go for it! Full documentation on template hierarchy is here: https://developer.wordpress.org/themes/basics/template-hierarchy/